py/picca/utils.py

```diff
@@ -727,9 +727,9 @@ def unred(wave, ebv, R_V=3.1, LMC2=False, AVGLMC=False):
 
     # Compute optical portion of A(lambda)/E(B-V) curve
     # using cubic spline anchored in UV, optical, and IR
-    xsplopir = sp.concatenate(([0],10000.0/sp.array([26500.0,12200.0,6000.0,5470.0,4670.0,4110.0])))
-    ysplir = sp.array([0.0,0.26469,0.82925])*R_V/3.1
-    ysplop = sp.array((sp.polyval([-4.22809e-01, 1.00270, 2.13572e-04][::-1],R_V ),
+    xsplopir = sp.concatenate(([0],10000.0/np.array([26500.0,12200.0,6000.0,5470.0,4670.0,4110.0])))
```

Comment:
should we use numpy concatenate?

Reply:
This is one of the example that no warning were trigered by the tests. So either it is not tested, or scipy does not give a warning. I would say that it is un safe to change if it is not tested. What do you think ?

Reply:
I'd probably leave it for now. May be we can start an Issue about removing Scipy when not necessary, but this is not a priority if they don't cause warnings.